app/src/main/java/com/kms/cura/view/fragment/PatientHomeFragment.java

```diff
@@ -280,6 +294,36 @@ public void callBackReload() {
         setUpSpnSpeciality();
     }
 
+    @Override
+    public void handleEvent(String event, String data) {
```

Comment:
Split this method into sub-methods. For ex:
- searchSuccess()
- loginFailed()
- connectionError()
...